app/models/observation_accuracy_experiment.rb

```diff
@@ -336,15 +465,63 @@ def generate_sample
     puts "Sample generated in #{duration} seconds."
   end
 
-  def contact_validators( validator_deadline_date: 1.week.from_now.strftime( "%Y-%m-%d" ) )
+  def observation_accuracy_validator_contact( validator )
+    return false unless ( user = User.where( id: validator.user_id ).first )
+
+    admin = User.where( email: "admin@inaturalist.org" ).first
```

Comment:
This might be better set up as a config variable. You're looking to load a particular iNaturalist user by email, and maybe the email to use for lookup should be defined in config.yml and the config value used here to look it up, rather than hardcoding the email in this model. We don't reference this email address anywhere else in this codebase

Reply:
Thanks - I'll slack you about how to make something like CONFIG.admin_user_email available